contrib/scriptlets/deb/postinst

```diff
@@ -26,7 +26,8 @@ helpMessage="If you need help using the app, use the command 'nordvpn --help'."
 
 # update configuration and shared library cache for linker to find .so files
 echo "/usr/lib/nordvpn" > /etc/ld.so.conf.d/nordvpn.conf
-ldconfig
+# NOTE: sudo is required here even though the script is run with EUID 0
+sudo ldconfig
```

Comment:
Shouldn't be executed with `sudo` also in postrm?

Reply:
Didn't notice anything wrong on postrm. I can add it "just in case"